emu/float80-test.c

```diff
@@ -5,6 +5,9 @@
 #include <fenv.h>
 #include "float80.h"
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wliteral-range"
```

Comment:
Is this a clang-only warning? If not, use GCC pragmas. If so, I don't see any need to push when you just pop at the end of the file